manager/ui/hawtio/plugins/api-manager/ts/catalog/api-catalog.ts

```diff
@@ -60,6 +60,9 @@ module Apiman {
                     if (api.endpointType == 'soap') {
                         api.ticon = 'fa-file-code-o';
                     }
+                    if (api.routeDefinitionUrl != null) {
+                        api.definitionUrl = api.routeDefinitionUrl;
+                    }
                 });
```

Comment:
No point in fabric8, but it might make sense on other platforms that don't support routes all together?